Assets/MixedRealityToolkit.SDK/Experimental/Features/Boundary/Scripts/BoundarySystemManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Boundary
+{
+    /// <summary>
+    /// Service manager supporting running the boundary system, without requring the MixedRealityToolkit object.
+    /// </summary>
+    public class BoundarySystemManager : BaseServiceManager
+    {
+        [SerializeField]
+        [Tooltip("The scale (room, world, etc) of the experience.")]
+        private ExperienceScale expereinceScale = ExperienceScale.Room;
```

Comment:
>ExperienceScale [](start = 16, length = 15)

Also, do you know why this is on this, and not the profile itself? I see that on the regular MRTK object it's also on the main thing, but it's really just used for the boundary stuff.

Reply:
It was originally intended to span multiple systems. Currently, it only impacts boundary, though it can (should) be used to determine the location of the origin (head or floor)